tests/util.py

```diff
@@ -1,7 +1,8 @@
+from typing import Optional
 from unittest.mock import Mock
 
 
-def get_test_json(name: str) -> dict or None:
+def get_test_json(name: str) -> Optional[dict]:
```

Comment:
Does it have any benefit to use `Optional`? Imo the `or` variant is way more intuitive to read.

Reply:
mypy reported the `or` variant is invalid.

Reply:
Works in pycharm, but so be it.